.gitignore

```diff
@@ -257,3 +257,6 @@ pub/
 
 #Ignore marker-file used to know which docker files we have.
```

Comment:
Please update the application to pull the connection strings from keyvault instead of appsettings.  That will eliminate the need to manually update the appsettings.json file after provision.